src/System.Private.ServiceModel/tests/Scenarios/Binding/Tcp/NetTcpBindingTests.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
```

Comment:
Reviewers: FYI -- this is unrelated to my PR but ToF could not compile this file without this due to the new use of Console.WriteLine (in ToF-only conditions).  So I'm fixing it as part of this PR to allow testing.